VideoCaptureSample/MainPage.xaml

```diff
@@ -26,25 +26,26 @@
     xmlns:local="using:VideoCaptureSample"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    mc:Ignorable="d">
+    mc:Ignorable="d" Width="640" Height="480">
```

Comment:
why is this hard coded to 640x480?  do we need to set values for width and height?